From 19ee7b070aff404018e65b511a6ace360b9a0557 Mon Sep 17 00:00:00 2001
From: Bin Meng <bmeng.cn@gmail.com>
Date: Wed, 6 Apr 2022 21:37:42 +0800
Subject: [PATCH 1/6] makefile: Drop "-g" compiler flag for mingw* builds

At present the makefiles for mac/mingw* builds have the "-g" compiler
flag, while other platform builds don't.

Let's drop it given "-g" is only useful for a DEBUG build.

Signed-off-by: Bin Meng <bmeng.cn@gmail.com>
---
 Makefile.mingw64 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.mingw64 b/Makefile.mingw64
index 7e4b32b..4cd1be9 100644
--- a/Makefile.mingw64
+++ b/Makefile.mingw64
@@ -2,7 +2,7 @@ CC=/usr/bin/x86_64-w64-mingw32-gcc
 CXX=/usr/bin/x86_64-w64-mingw32-g++
 STRIP=/usr/bin/x86_64-w64-mingw32-strip
 CFLAGS=-O2 -Wall -static -static-libgcc -static-libstdc++  -D_FILE_OFFSET_BITS=64 -g
-CXXFLAGS=-O2 -Wall -static -static-libgcc -static-libstdc++ -D_FILE_OFFSET_BITS=64 -g
+CXXFLAGS=-O2 -Wall -static -static-libgcc -static-libstdc++ -D_FILE_OFFSET_BITS=64
 #CXXFLAGS=-O2 -Wall -D_FILE_OFFSET_BITS=64 -I /usr/local/include -I/opt/local/include -g
 LIB_NAMES=guid gptpart bsd parttypes attributes crc32 mbrpart basicmbr mbr gpt support diskio diskio-windows
 MBR_LIBS=support diskio diskio-windows basicmbr mbrpart
-- 
2.35.1

